source/includes/_dataset_resources.md

```diff
@@ -0,0 +1,276 @@
+# Dataset Resources
+
+## The resource object
+
+The resource object describe a resource on the OpenDataSoft platform. It is composed of a URL, a title, a type, a parameter object and a credentials object. The URL is where data will be pulled to populate the dataset. Resources urls can (and often do!) point to [files](#files) uploaded to the platform using the `odsfile://` url scheme.
```

Comment:
The resource object _describes_ a resource

Reply:
:+1: